api/v1alpha1/dopplersecret_types.go

```diff
@@ -82,6 +82,10 @@ type DopplerSecretSpec struct {
 	// +optional
 	Config string `json:"config,omitempty"`
 
+	// A list of secrets to sync from the config
+	// +optional
+	Secrets []string `json:"secrets,omitempty"`
```

Comment:
What do you think about calling this field `onlySecrets` to match the phrasing `doppler run --only-secrets`?

Reply:
We could do that. Then we'd not be matching the phrasing we use in the API though (the parameter is literally called `secrets`). 🤷‍♂️

Reply:
I trust your sensibilities here. If you think it's clear enough to have `secrets` alongside `manageSecret` and `tokenSecret`, I think we're set

Reply:
I think it'll be pretty clear. Also, `--only-secrets` has always rubbed me a little wrong – makes me think of `--only-names` which has completely different behavior.